crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -1074,11 +1082,787 @@ pub mod tests {
         );
         server_builder.build_schema();
 
-        let resp = reqwest::get(&url).await.unwrap();
-        assert_eq!(resp.status(), reqwest::StatusCode::OK);
+        // let resp = reqwest::get(&url).await.unwrap();
+        // assert_eq!(resp.status(), reqwest::StatusCode::OK);
```

Comment:
should we remove these?

Reply:
Good spot -- no, I think Stefan commented these out because the test is flaky, but actually this is now fixed in `main`, so I'll put this back and rebase over the proper fix.